README.md

```diff
@@ -629,6 +629,151 @@ Can you try updating the rendering function to also show `entry.content` and
 > You might need to use `switch` and the `dangerouslySetInnerHTML` prop for
 > `entry.content`, and `Array.map` for `entry.links`.
 
+## Step 8: Using input fields to get data for any user
+
+In this step, we'll enhance our app by letting users enter a GitHub username to
+fetch and display their activity feed.
+
+To do this, we will have to update our app in a few ways.
```

Comment:
Worth adding the "product" idea?

To do this, we will have to update our app in a few ways: `add an input for the username, store the username, fire the request to Github and react will display the results`

Reply:
added in https://github.com/ahrefs/react-alicante-workshop/pull/6/commits/1e63afcbe33c61d6e38e696b507d10400a80f14b.